app/controllers/noticeboards_controller.rb

```diff
@@ -6,9 +6,15 @@ class NoticeboardsController < ApplicationController
   before_action :init_graphql_client
 
   def index
+    @noticeboard_name = noticeboard_params[:name] || "Schwarzes Brett"
+    category_ids = noticeboard_params[:category_ids]
+
     results = @smart_village.query <<~GRAPHQL
       query {
-        genericItems(genericType: "Noticeboard") {
+        genericItems(
+          genericType: "Noticeboard",
+          #{category_ids.present? ? "categoryIds: #{category_ids}" : ""}
```

Comment:
did not know you can include ruby in <<~ Strings

Reply:
i also got help for this ;)